util/update/run.go

```diff
@@ -49,5 +52,29 @@ func Run(path string) error {
 	// update the model
 	update := newUpdate()
 
+	newVersion := getCurrentVersion(path)
+
+	fmt.Printf("Updated to version: %s", newVersion)
+
+	printGithubReleaseUrl(newVersion)
+
 	return update.Save()
 }
+
+func getCurrentVersion(path string) string {
+	if path == "" {
+		fmt.Errorf("invalid path")
+	}
+	version, err := exec.Command(path, "version").Output()
+	if err != nil {
+		fmt.Errorf("Error while trying to get the nanobox version")
+		return ""
+	}
+	return string(version)
+}
+
+func printGithubReleaseUrl(version string) {
```

Comment:
Linking to the more up-to-date changelog would eliminate the need for this function.